src/script/ceph-backport.sh

```diff
@@ -73,6 +73,11 @@
 # git cherry-pick -x ...
 # ceph-backport.sh 19206 jewel
 #
+# optionally, you can set the component label that will be added to the PR with
+# an environment variable:
+#
+# component=dashboard ceph-backport.sh 19206 jewel
```

Comment:
nit: as an old-school guy, I prefer all-caps for environment variables. Could we change this to:

```
# COMPONENT=dashboard ceph-backport.sh 19206 jewel
```

?

Reply:
Sure, only used lowercase to use the same var name as the one we already had.